source/common/quic/platform/BUILD

```diff
@@ -149,6 +149,9 @@ envoy_quiche_platform_impl_cc_library(
 envoy_quiche_platform_impl_cc_library(
     name = "quiche_export_impl_lib",
     hdrs = ["quiche_export_impl.h"],
-    external_deps = ["abseil_base"],
+    external_deps = [
+        "abseil_base",
+        "ssl",
```

Comment:
I'm still puzzled why QUICHE_EXPORT_IMPL needs to depend on boring SSL?

Reply:
It's because of this commit https://quiche.googlesource.com/quiche/+/fc95ef694198fe6cafb482086414fa9406be1bae

Reply:
I might be missing something, but it looks like quiche_export_impl.h is pretty self contained and I would not expect it to be affected by that QUICHE commit:

```
#pragma once

// NOLINT(namespace-envoy)

// This file is part of the QUICHE platform implementation, and is not to be
// consumed or referenced directly by other Envoy code. It serves purely as a
// porting layer for QUICHE.

#include "absl/base/attributes.h"

// These macros are documented in: quiche/quic/platform/api/quic_export.h

#if defined(_WIN32)
#define QUICHE_EXPORT_IMPL
#elif ABSL_HAVE_ATTRIBUTE(visibility)
#define QUICHE_EXPORT_IMPL __attribute__((visibility("default")))
#else
#define QUICHE_EXPORT_IMPL
#endif

#define QUICHE_EXPORT_PRIVATE_IMPL QUICHE_EXPORT_IMPL
#define QUICHE_NO_EXPORT_IMPL QUICHE_EXPORT_IMPL
```

Reply:
Uhh I see why my change was surprising.
I should add that external_deps to the actual `quiche_common_platform_default_quiche_platform_impl_time_utils_impl_lib` target instead of the common one.
done